dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/bundle/TrackedEntityAttributeTest.java

```diff
@@ -70,6 +73,7 @@ protected void initTest()
         throws IOException
     {
         setUpMetadata( "tracker/te_with_tea_metadata.json" );
+        injectAdminUser();
```

Comment:
is it necessary/a good idea to use an admin user in our tests?

Reply:
I think this is a big question. And I agree that we should deal with the whole users mechanism in a much smarter way in the tests.
But I think we should discuss about it when we discuss a little more in depth on test strategy.
At the moment, we have almost all integration tests using admin users.
We select a different user only if we are really testing the permissions part.